util/state_division_to_dcid.py

```diff
@@ -0,0 +1,167 @@
+    "Nevada": "geoId/32",
+    "New Mexico": "geoId/35",
+    "Utah": "geoId/49",
+    "Wyoming": "geoId/56",
+    "Pacific": "usc/PacificDivision",
+    "Alaska": "geoId/02",
+    "California": "geoId/06",
+    "Hawaii": "geoId/15",
+    "Oregon": "geoId/41",
+    "Washington": "geoId/53",
+    "U.S. Territories": "",
+    "American Samoa": "geoId/60",
+    "Commonwealth of Northern Mariana Islands": "geoId/69",
+    "Guam": "geoId/66",
+    "Puerto Rico": "geoId/72",
+    "U.S. Virgin Islands": "geoId/78",
+    "US Virgin Islands": "geoId/78",
+    "Non-U.S. Residents": "country/USA",
+    "Total": "",
+    "MultiState": "",
+    "Multistate": "",
+    "UNITED STATES": "country/USA",
+    "NEW ENGLAND": "usc/NewEnglandDivision",
+    "Conn.": "geoId/09",
+    "Mass.": "geoId/25",
+    "N.H.": "geoId/33",
+    "R.I.": "geoId/44",
+    "Vt.": "geoId/50",
+    "MID. ATLANTIC": "usc/MiddleAtlanticDivision",
+    "N.J.": "geoId/34",
+    # geoId/36 is not used for resolution since it was unclear whether upstate New York was New York State (geoId/36) + New York City (geoId/3651000)
+    "N.Y. (Upstate)": "",
+    # geoId/3651000 is not used for resolution because the boundaries of NY City was unclear in the dataset's context
+    "N.Y. City": "",
+    "Pa.": "geoId/42",
+    "E.N. CENTRAL": "usc/EastNorthCentralDivision",
+    "Ill.": "geoId/17",
+    "Ind.": "geoId/18",
+    "Mich.": "geoId/26",
+    "Wis.": "geoId/55",
+    "W.N. CENTRAL": "usc/WestNorthCentralDivision",
+    "Kans.": "geoId/20",
+    "Minn.": "geoId/27",
+    "Mo.": "geoId/29",
+    "Nebr.": "geoId/31",
+    "N. Dak.": "geoId/38",
+    "S. Dak.": "geoId/46",
+    "S. ATLANTIC": "usc/SouthAtlanticDivision",
+    "Del.": "geoId/10",
+    "D.C.": "geoId/11",
+    "Fla.": "geoId/12",
+    "Ga.": "geoId/13",
+    "Md.": "geoId/24",
+    "N.C.": "geoId/37",
+    "S.C.": "geoId/45",
+    "Va.": "geoId/51",
+    "W. Va.": "geoId/54",
+    "E.S. CENTRAL": "usc/EastSouthCentralDivision",
+    "Ala.": "geoId/01",
+    "Ky.": "geoId/21",
+    "Miss.": "geoId/28",
+    "Tenn.": "geoId/47",
+    "W.S. CENTRAL": "usc/WestSouthCentralDivision",
+    "Ark.": "geoId/05",
+    "La.": "geoId/22",
+    "Okla.": "geoId/40",
+    "Tex.": "geoId/48",
+    "MOUNTAIN": "usc/MountainDivision",
+    "Ariz.": "geoId/04",
+    "Colo.": "geoId/08",
+    "Mont.": "geoId/30",
+    "Nev.": "geoId/32",
+    "N. Mex.": "geoId/35",
+    "Wyo.": "geoId/56",
+    "PACIFIC": "usc/PacificDivision",
+    "Calif.": "geoId/06",
+    "Oreg.": "geoId/41",
+    "Wash.": "geoId/53",
+    "Amer. Samoa": "geoId/60",
+    "C.N.M.I.": "geoId/69",
+    "Commonwealth of the Northern Mariana Islands": "geoId/69",
+    "P.R.": "geoId/72",
+    "Republic of the Marshall Islands": "country/MHL",
+    "Republic of Palau": "country/PLW",
+    "V.I.": "geoId/78"
+}
+
+
+def get_place_dcid(place_name: str) -> str:
+    """
+  Utility wrapper that returns the place dcid given a name
+  """
+    return _PLACE_MAP[place_name]
```

Comment:
Since this is a utility, better to avoid KeyError exception:
return _PLACE_MAP.get(place_name, '')

Reply:
This is a nice suggestion, thank you! I have made this change.